exchanges/btcmarkets/btcmarkets_websocket.go

```diff
@@ -205,3 +228,21 @@ func (b *BTCMarkets) Subscribe(channelToSubscribe wshandler.WebsocketChannelSubs
 	}
 	return b.WebsocketConn.SendMessage(req)
 }
+
+// Login logs in allowing private ws events
+func (b *BTCMarkets) generateAuthSubscriptions() error {
+	var authSub WsAuthSubscribe
+	signTime := strconv.FormatInt(time.Now().UTC().UnixNano()/1000000, 10)
+	strToSign := "/users/self/subscribe" + "\n" + signTime
+	tempSign := crypto.GetHMAC(crypto.HashSHA512,
+		[]byte(strToSign),
+		[]byte(b.API.Credentials.Secret))
+	sign := crypto.Base64Encode(tempSign)
+	authSub.MarketIDs = []string{"BTC-AUD"}
```

Comment:
Please get the enabled pairs and use them instead of hard coding just this singular currency